core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.ts

```diff
@@ -254,10 +254,13 @@ export class CustomizeInteractionModalComponent
   }
 
   isSaveInteractionButtonEnabled(): boolean {
+    let exploration = this.getTitle(this.stateInteractionIdService.displayed);
     return !!(
-      this.hasCustomizationArgs &&
-      this.stateInteractionIdService.displayed &&
-      (this.getCustomizationArgsWarningsList().length === 0));
+      exploration === 'End Exploration' ||
+      (
+        this.hasCustomizationArgs &&
+        this.stateInteractionIdService.displayed &&
+        this.getCustomizationArgsWarningsList().length === 0));
   }
```

Comment:
```suggestion
      if (explorationTitle === 'End Exploration') {
        return true;
      }
      return !!(
        this.hasCustomizationArgs &&
        this.stateInteractionIdService.displayed &&
        this.getCustomizationArgsWarningsList().length === 0
      );
  }
```

Reply:
Also should the button be enabled even if `this.getCustomizationArgsWarningsList().length != 0` for 'End Exploration'?

Reply:
Yes it can be enabled whether the length of "getCustomizationArgsWarningsList" is zero or not.